src/org/zaproxy/zap/authentication/AuthenticationMethodType.java

```diff
@@ -149,22 +151,23 @@ public abstract AuthenticationMethod loadMethodFromSession(Session session, int
 	 * @param authMethod the auth method to persist
 	 * @throws UnsupportedAuthenticationMethodException the unsupported authentication method
 	 *             exception
+	 * @throws DatabaseException if an error occurred while persisting the authentication method
 	 */
 	public abstract void persistMethodToSession(Session session, int contextId,
-			AuthenticationMethod authMethod) throws UnsupportedAuthenticationMethodException, DatabaseException;
+			AuthenticationMethod authMethod) throws DatabaseException;
 
 	/**
 	 * Export the specified method to the configuration
-	 * @param config
-	 * @param authMethod
+	 * @param config the configurations where to export/save the authentication method
```

Comment:
Yes, the data of the authentication method is "saved" to the configuration.
Would the following be more clear?
`the {@code Configuration} where to export/save the authentication method`